src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1434,11 +1444,14 @@ const BasicUserInfo: React.FC<{
     let verifyButton;
     const homeserverSupportsCrossSigning = useHomeserverSupportsCrossSigning(cli);
 
-    const userTrust = cryptoEnabled && cli.checkUserTrust(member.userId);
-    const userVerified = cryptoEnabled && userTrust && userTrust.isCrossSigningVerified();
+    const userTrust = useAsyncMemo<UserVerificationStatus | undefined>(
+        async () => cli.getCrypto()?.getUserVerificationStatus(member.userId),
+        [member.userId],
+    );
```

Comment:
will this correctly update if the verification status is updated while the userinfo is open? (cf https://github.com/vector-im/element-web/issues/26279)

Reply:
Are you talking about this use case ? 

https://github.com/matrix-org/matrix-react-sdk/assets/2621378/2f7550c9-58c1-498a-99f9-f48b2ea4435a

(If I refresh, the verify button is not visible)

Reply:
Not quite. Suppose you have two sessions, let's say A, B. In session A you open Bob's userinfo which shows "unverified". In session B you *verify* Bob. Once you complete verification, does the open userinfo in session A update?

(This may turn out to be Difficult, in which case feel free to park it and open a new issue)

Reply:
Right.

Create an issue about it https://github.com/vector-im/element-web/issues/26314